contracts/libs/Owned.sol

```diff
@@ -13,5 +13,9 @@ contract Owned {
 
   function isOwner(address addr) public returns(bool) { return addr == owner; }
 
-  function transfer(address _owner) onlyOwner { owner = _owner; }
+  function transfer(address _owner) onlyOwner {
+    if (_owner != address(this)) {
```

Comment:
Are you sure this check is worthwhile? It seems more likely that a user is going to input a wrong address, as compared to the address of the controller, but we can hardly check for that.